### PR TITLE
@W-8542557 - flawed search string function fixed

### PR DESCRIPTION
**Pull request by kritikapradhan**

### CODE REVIEW

File: pkg/sloop/webserver/webfiles.go [hunk 1]
```diff
@@ -52,3 +51,12 @@ func getTemplate(templateName string, _ []byte) (*template.Template, error) {
 	}
 	return newTemplate, nil
 }
+
+func contains(list []string, elem string) bool {
```
Comment (sana-jawad) [inline pkg/sloop/webserver/webfiles.go:55]: Please add unit tests for this function

**Review by sana-jawad — CHANGES_REQUESTED** [review]
Looks good. Please add unit tests

File: pkg/sloop/webserver/webfiles.go (hunk 1 above)
Comment (kritikapradhan) [reply pkg/sloop/webserver/webfiles.go]: review pls

[the remark this answers is not in the document]
Comment (kritikapradhan) [reply pkg/sloop/webserver/webfiles.go]: review pls

**Review by mallow111 — DISMISSED** [review]

File: pkg/sloop/webserver/webfiles_test.go [hunk 2]
```diff
@@ -46,3 +49,15 @@ func writeFile(t *testing.T, fs *afero.Afero, filePath string, content string) {
 	err = fs.WriteFile(filePath, []byte(content), defaultFileMode)
 	assert.Nil(t, err)
 }
+
+func Test_FileExistsInList_True(t *testing.T) {
```
Comment (sana-jawad) [inline pkg/sloop/webserver/webfiles_test.go:53]: Similarly move these to https://github.com/salesforce/sloop/blob/master/pkg/sloop/common/utilities_test.go

File: pkg/sloop/webserver/webfiles.go [hunk 3]
```diff
@@ -52,3 +51,12 @@ func getTemplate(templateName string, _ []byte) (*template.Template, error) {
 	}
 	return newTemplate, nil
 }
+
+func contains(fileList []string, elem string) bool {
```
Comment (sana-jawad) [inline pkg/sloop/webserver/webfiles.go:55]: If we make this generic please rename to stringList

**Review by sana-jawad — CHANGES_REQUESTED** [review]
Minor comments

**Review by sana-jawad — APPROVED** [review]

**Pull request merged**